ReSharper.FSharp/test/data/parsing/_.fs

```diff
@@ -1,10 +1,4 @@
 namespace global
 
-type T() =
```

Comment:
Please don't update this file.

I should check if we can ignore it after it's already been checked in.

Reply:
Reverted the changes here